src/bundle.c

```diff
@@ -1095,22 +1272,28 @@ int install_bundles_frontend(char **bundles)
 	}
 
 	timelist_timer_start(global_times, "Prepend bundles to list");
+	aliases = get_alias_definitions();
 	for (; *bundles; ++bundles) {
-		bundles_list = list_prepend_data(bundles_list, *bundles);
-		if (*bundles) {
+		struct list *alias_bundles = get_alias_bundles(aliases, *bundles);
```

Comment:
I'm a bit concerned about if we should let system aliases override bundles that already exists.
A bundle emacs, for example could install a file creating an alias "vim emacs" and then vim bundle will never be installed. And there's nothing vim maintainers can do about that.

If a user override that, it's ok, but a package maintainer shouldn't

Reply:
I'm inclined to say that is an content OS policy decision rather than update tool decision and that it can be tested against at that level if it is desired.

That said I may also be not excited to change this because doing a lookup to see if an alias defined is a bundle when creating definitions didn't seem particularly appealing to me when I was writing the code X(.

Reply:
> I'm inclined to say that is an content OS policy decision rather than update tool decision and that it can be tested against at that level if it is desired.

I took some time to think about that yesterday and I concluded you are right about that. There are so many other ways a package maintainer could create problems to other packages (create invalid configuration or library files, for example. The OS maintainers should be in charge to make sure this won't happen.

So, ignore my comment and leave this the way it is.